docs/resources/ui-building-blocks/components/built-in-components/pincode.md

```diff
@@ -0,0 +1,339 @@
+
+![Aligning pin code fields](../../../control-flow/user-interactivity/imgs/align-pin-code-fields.gif)
+
+### Changing pin field shape and size
+
+To change the pin field shape and size:
+
+1. Select the **PinCode** widget, move to the properties panel, find the **Pin Field Shape** property, and here you can set the shape to **Box**, **Circle**, and **Underline**.
+2. To change the height and width, enter the value in **Field Height**, and **Field Width** boxes.
+3. To create a rounded border when the shape is set to *Box*, use the **Border Radius** and **Border Width** properties.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://www.loom.com/embed/fa11e65c9c0a413fbae2f271bc9d4333?sid=77905cea-5db6-4d7b-8927-d70f0daaa3ed"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+### Change colors
+You can change colors for the different states of the pin fields. To do so:
+
+1. Select the **PinCode** widget, move to the properties panel, and change the colors for the following properties:
+    - **Active Color**: This sets the border color when the value is entered.
+    - **Inactive Color**: This sets the border color when there is no value.
+    - **Selected Color**: This sets the border color when the cursor is inside the pin field and the user is about to enter the value.
+2. To change the background color instead of only the border color, **Enable Active Fill**.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://www.loom.com/embed/d8ccff9efd844c50a006b5004f09fd15?sid=f568dd9e-3adc-45c3-a6d5-88cdfb0b76db"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+### Customizing cursor
+
+You can show/hide the cursor using the **Show Cursor** toggle and change the color using the **Cursor Color** property.
+
+### Clear pin code value
+
+See how to [reset the pin code value](#).
```

Comment:
Where is this link supposed to go? 

And same comment as dropdown about putting them in an admonition.

Reply:
Added!